### PR TITLE
Add Avalon8 support

### PR DESCRIPTION
**Pull request by XzxEmbedded**

### CODE REVIEW

File: scripts/build-avalon-image.sh [hunk 1]
```diff
@@ -229,7 +232,7 @@ Usage: $0 [--version] [--help] [--build] [--cgminer] [--cleanup]
                         use pi-modelb-v2 if unset
 
      AVA_MACHINE        Environment variable, available machine:
-                        avalon7, avalon6, avalon4
+                        avalon8, avalon7, avalon6, avalon4
                         use avalon6 if unset
 
 Written by: Xiangfu <xiangfu@openmobilefree.net>
```
Comment (qinfengling) [inline scripts/build-avalon-image.sh]: @XzxEmbedded Append your name please.

**Comment by XzxEmbedded** [comment]
Append my name and email, checking it please, thanks.

**Comment by XzxEmbedded** [comment]
Update the indentation. Thanks.

**Comment by qinfengling** [comment]
@XzxEmbedded Great. Thanks.

**Pull request merged**